example/flask/server.py

```diff
@@ -100,16 +100,15 @@ def crashcallback():
 
 
 @app.route('/handled')
-def notify():
+def handle_zero_div():
     """Deliberately triggers a handled exception, and reports it to Bugsnag.
     """
     try:
         x = 1/0
-    except ZeroDivisionError:
-        bugsnag.notify(ZeroDivisionError('Flask demo: To infinity... and beyond!'))
```

Comment:
This change makes this particular demo inconsistent with the rest of the examples (Django 2, Django 1, celery). What's the justification for that (and how is that related to the runtime version)?

Reply:
It wasn't related to runtime version directly; it was related to a previous comment in this pull request (https://github.com/bugsnag/bugsnag-python/pull/182#discussion_r294985483) for the new bottle example. I would have preferred that similar changes to different example apps be separate changes or pull requests, but from my previous jobs/companies, fighting every battle that I wanted was too mentally exhausting and lead to me being heavily criticized as "too picky" (and other similar things), so I had to pick my battles.

Reply:
I agree with our comments @tobyhs on #182 - the examples are already a little inconsistent and break style rules, so I've raised a ticket to sort this out separately.